shortlister/view.py

```diff
@@ -20,11 +20,20 @@ def view_applicant_details(self, applicant: Applicant):
         """Prints details of applicant to console."""
         print(f"Details for {applicant.name}: ")
         print(f"CV Path: {applicant.cv}")
-        print("Scores:")
-        if applicant.scores is not None:
+        print("Scores:", end=" ")
+
+        if applicant.scores == {}:
+            print("No scores")
+        else:
+            print()
```

Comment:
In Python, a dictionary evaluates to `False` in a condition if empty and `True` otherwise. So, a more "Pythonic" way is

```
if applicant.scores:
    print("Scores:", end=" ")
    etc....
else:
    print("No scores")
```

Reply:
Is it generally better to first check if a value is True rather than if it is empty/None or False?

Reply:
It depends on the situation. If your variable holding the dictionary could possibly be `None`, this won't work. But we initialise Applicant with empty dictionary, so here it's clear.